src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -230,25 +290,26 @@ function PaymentMethodList(props) {
                 data={filteredPaymentMethods}
                 renderItem={renderItem}
                 keyExtractor={(item) => item.key}
-                ListEmptyComponent={renderListEmptyComponent(translate)}
-                ListHeaderComponent={props.listHeaderComponent}
-                onContentSizeChange={props.onListContentSizeChange}
+                ListEmptyComponent={shouldShowEmptyListMessage ? renderListEmptyComponent(translate) : null}
```

Comment:
why `renderListEmptyComponent` is being called and passing `translate` when the function uses the `translate` from props and not from the function params?

Also I don't see the need to wrap `renderListEmptyComponent` in a `useCallback`